com.google.AndroidStudio.json

```diff
@@ -47,12 +47,12 @@
                 {
                     "type": "extra-data",
                     "filename": "android-studio.tar.gz",
-                    "size": 925388022,
+                    "size": 984074364,
                     "only-arches": [
                         "x86_64"
                     ],
-                    "url": "https://dl.google.com/dl/android/studio/ide-zips/4.1.3.0/android-studio-ide-201.7199119-linux.tar.gz",
-                    "sha256": "f599749ca47cda06d392e2764017c8a8a0c7b963a6a88ed494b432bece7cbc1b"
+                    "url": "https://redirector.gvt1.com/edgedl/android/studio/ide-zips/2020.3.1.14/android-studio-2020.3.1.14-linux.tar.gz",
```

Comment:
Could you use the url?:
https://dl.google.com/dl/android/studio/ide-zips/2020.3.1.14/android-studio-2020.3.1.14-linux.tar.gz

Thank you for your contribution.

Reply:
Sure I can try that. I thought that URL from the canary download site was weird, too...

Reply:
updated :)